flow_pre_lda.scala

```diff
@@ -134,18 +142,139 @@ def bin_column(value: String, cuts: Array[Double]) = {
 }
 
 
+def convert_feedback_row_to_flow_row(feedBackRow: String) = {
```

Comment:
I would definitely prefer that we use the column name approach that is used in DNS pipeline over this.  It would reduce code and also be more robust to changes in column order